code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -245,12 +256,15 @@
 
 			var/heat = ((1 - cold_protection) * 0.1 + conduction_coefficient) * temperature_delta * (air_heat_capacity * heat_capacity / (air_heat_capacity + heat_capacity))
 
-			air1.temperature = max(air1.temperature - heat / air_heat_capacity, TCMB)
+			air1.temperature = clamp(air1.temperature - heat / air_heat_capacity, TCMB, MAX_TEMPERATURE)
 			mob_occupant.adjust_bodytemperature(heat / heat_capacity, TCMB)
 
 		air1.gases[/datum/gas/oxygen][MOLES] = max(0,air1.gases[/datum/gas/oxygen][MOLES] - 0.5 / efficiency) // Magically consume gas? Why not, we run on cryo magic.
 		air1.garbage_collect()
 
+		if(air1.temperature > 2000)
+			take_damage(clamp((air1.temperature)/300, 7, 15), BURN, 0, 0)

```

Comment:
```suggestion
			take_damage(clamp((air1.temperature)/300, 7, 15), BURN, sound_effect = FALSE)
```
Third argument is a string, fourth is a boolean.

Reply:
i'll just remove those then, leaving only the amount and the damage type